src/api/spec/features/webui/maintenance_workflow_spec.rb

```diff
@@ -0,0 +1,172 @@
+require 'browser_helper'
+# WARNING: If you change tests make sure you uncomment this line
+# and start a test backend. Some of the BsRequestAction methods
+# require real backend answers for projects/packages.
+# CONFIG['global_write_through'] = true
+
+RSpec.feature 'Projects', type: :feature, js: true do
+  let(:admin_user) { create(:admin_user) }
+  let(:user) { create(:confirmed_user, login: 'tom') }
+  let(:maintenance_coord_user) { create(:confirmed_user, login: 'maintenance_coord') }
+  let(:project) { create(:project_with_repository, name: 'ProjectWithRepo') }
+  let(:package) { create(:package_with_file, project: project, name: 'ProjectWithRepo_package') }
+  let!(:maintained_attrib) { create(:maintained_attrib, package: package) }
+  let(:update_project) { create(:update_project, target_project: project, name: "#{project.name}:Update") }
+  let(:maintenance_project) {
+    create(:maintenance_project,
+           name: 'MaintenanceProject',
+           title: 'official maintenance space',
+           target_project: update_project,
+           create_patchinfo: true,
+           maintainer: maintenance_coord_user)
+  }
+
+  scenario 'maintenance workflow' do
+    # admin user add attribute 'OBS:Maintained' to project
+    login(admin_user)
+
+    visit project_show_path(project)
+
+    click_link('Advanced')
+    click_link('Attributes')
+    click_link('Add a new attribute')
+    select('OBS:Maintained', from: 'attrib_attrib_type_id')
+    click_button('Create Attribute')
+    expect(page).to have_text('Attribute was successfully created.')
+
```

Comment:
Don't do this via clicking please. Create a Maintained factory and setup everything up front.